<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>472</width>
    <height>447</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:#444444;</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QListWidget" name="listWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>471</width>
      <height>192</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>30</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #333333;
color:white;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushBackBut">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>260</y>
      <width>130</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: #333333;
    color: #ffffff;
    font-family: &quot;Arial Bold&quot;;
    font-size: 20px;
    border-radius: 10px;
    
}

QPushButton:hover {
    background-color: #444444;
}

QPushButton:pressed {
    background-color: #222222;
}</string>
    </property>
    <property name="text">
     <string>PushBack</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushFrontBut">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>260</y>
      <width>130</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: #333333;
    color: #ffffff;
    font-family: &quot;Arial Bold&quot;;
    font-size: 20px;
    border-radius: 10px;
    
}

QPushButton:hover {
    background-color: #444444;
}

QPushButton:pressed {
    background-color: #222222;
}</string>
    </property>
    <property name="text">
     <string>PushFront</string>
    </property>
   </widget>
   <widget class="QPushButton" name="popBackBut">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>320</y>
      <width>130</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: #333333;
    color: #ffffff;
    font-family: &quot;Arial Bold&quot;;
    font-size: 20px;
    border-radius: 10px;
    
}

QPushButton:hover {
    background-color: #444444;
}

QPushButton:pressed {
    background-color: #222222;
}</string>
    </property>
    <property name="text">
     <string>PopBack</string>
    </property>
   </widget>
   <widget class="QPushButton" name="popFrontBut">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>320</y>
      <width>130</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: #333333;
    color: #ffffff;
    font-family: &quot;Arial Bold&quot;;
    font-size: 20px;
    border-radius: 10px;
    
}

QPushButton:hover {
    background-color: #444444;
}

QPushButton:pressed {
    background-color: #222222;
}</string>
    </property>
    <property name="text">
     <string>PopFront</string>
    </property>
   </widget>
   <widget class="QPushButton" name="clearBut">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>290</y>
      <width>130</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: #333333;
    color: #ffffff;
    font-family: &quot;Arial Bold&quot;;
    font-size: 20px;
    border-radius: 10px;
    
}

QPushButton:hover {
    background-color: #444444;
}

QPushButton:pressed {
    background-color: #222222;
}</string>
    </property>
    <property name="text">
     <string>Clear</string>
    </property>
   </widget>
   <widget class="QLabel" name="sizeLabel">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>200</y>
      <width>151</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>30</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:white;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>210</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>30</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:white;</string>
    </property>
    <property name="text">
     <string>Size:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="swapMinAndMax">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>200</y>
      <width>181</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: #333333;
    color: #ffffff;
    font-family: &quot;Arial Bold&quot;;
    font-size: 20px;
    border-radius: 10px;
    
}

QPushButton:hover {
    background-color: #444444;
}

QPushButton:pressed {
    background-color: #222222;
}</string>
    </property>
    <property name="text">
     <string>SwapMinAndMax</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>472</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
